### PR TITLE
Feat/store view

### PR DESCRIPTION
**Pull request by Madlenne**

### CODE REVIEW

File: frontend/src/components/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+    const handleChange = (event: any, newValue?: number | number[]) => {
+      setValue(newValue as number[]);
+    };
+
+    const onCategoryClick = (category: string) => {
+      history.push(`/insight${category}`)
+    }
+  
+    const handlePrice =(event: any, isMin: boolean) => {
```
Comment (gravvy23) [inline frontend/src/components/Sidebar/Sidebar.tsx:165]: I would turn it as 
```
const handlePrice = (isMin: boolean) => (event: any) => {
...
}

File: frontend/src/components/Sidebar/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+        <Price>PRICE</Price>
+        <Slider value={value} max={300} onChange={handleChange} aria-labelledby="continuous-slider" />
+        <div className={classes.inputs}>
+        <OutlinedInput
+            value={value[0]}
+            classes={{
+              root: classes.root,
+              input: classes.input,
+            }}
+            onChange={(event) => handlePrice(event, true)}
```
Comment (gravvy23) [inline frontend/src/components/Sidebar/Sidebar.tsx:184]: and then I would pass 
```
onChange={handlePrice(true)}
```

File: frontend/src/components/Sidebar/Sidebar.tsx [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+            }}
+          />
+          -
+          <OutlinedInput
+                value={value[1]}
+                classes={{
+                  root: classes.root,
+                  input: classes.input,
+                }}
+                onChange={(event) => handlePrice(event, false)}
```
Comment (gravvy23) [inline frontend/src/components/Sidebar/Sidebar.tsx:196]: the same here

File: frontend/src/components/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (gravvy23) [reply frontend/src/components/Sidebar/Sidebar.tsx:165]: btw why `any` ? :(

File: frontend/src/domain/Home/Catalogue/Catalogue.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            <Header>
+                Action
+            </Header>
+
+            <Items>
+                { items.map(({ title, categories, price, cover }) => <GameCard title={title} categories={categories} price={price} cover={cover}/>)}
+                
+            </Items>
+
+        </Container>
```
Comment (gravvy23) [inline frontend/src/domain/Home/Catalogue/Catalogue.tsx:49]: please, run some linter on changed files :)

File: frontend/src/components/GameCard/GameCard.tsx [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+    const classes = useStyles();
+
+    return <Container image={cover}>
+         <img src={cover} alt="Avatar" className={classes.image}/>
+         <Button variant="contained"  className={classes.button}>
+            Add to cart
+        </Button>
+        <Details>
+            <Title>{ title }</Title>
+            <Price>{ `${price} $` }</Price>
```
[the remark this answers is not in the document]
Comment (Madlenne) [reply frontend/src/components/GameCard/GameCard.tsx:116]: It's a syntax of string interpolation not prefix 😄

File: frontend/src/components/Sidebar/Sidebar.tsx (hunk 1 above)
Comment (Madlenne) [reply frontend/src/components/Sidebar/Sidebar.tsx:165]: 🤷‍♀️

File: frontend/src/components/GameCard/GameCard.tsx (hunk 5 above)
[the remark this answers is not in the document]
Comment (gravvy23) [reply frontend/src/components/GameCard/GameCard.tsx:116]: Oh God, right 😅

**Review by gravvy23 — APPROVED** [review]
Looks Good 👌🏻

**Comment by Madlenne** [comment]
@gravvy23 I did just a rebase ;)

**Comment by gravvy23** [comment]
force-push, dangerous 🤭

**Review by gravvy23 — APPROVED** [review]

**Comment by Madlenne** [comment]
It is needed after rebase because rebasing to the main branch changes commits hashes 😅

**Comment by gravvy23** [comment]
> It is needed after rebase because rebasing to the main branch changes commits hashes 😅

Yeah I know, that's why force-with-lease is better ;)

**Pull request merged**